### PR TITLE
max subcompactions

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
@BusyJay @siddontang PTAL

### CODE REVIEW

File: tests/test_rocksdb_options.rs [hunk 1]
```diff
@@ -203,3 +203,12 @@ fn test_pending_compaction_bytes_limit() {
...
     let db = DB::open(opts, path.path().to_str().unwrap()).unwrap();
 }
+
+#[test]
+fn test_set_max_subcompactions() {
+    let path = TempDir::new("_rust_rocksdb_max_subcompactions").expect("");
+    let mut opts = Options::new();
+    opts.create_if_missing(true);
+    opts.set_max_subcompactions(4);
+    let db = DB::open(opts, path.path().to_str().unwrap()).unwrap();
```
Comment (siddontang) [inline tests/test_rocksdb_options.rs]: db seems not be used, drop explicitly

**Comment by siddontang** [comment]
LGTM

**Comment by zhangjinpeng1987** [comment]
@BusyJay PTAL

**Comment by zhangjinpeng1987** [comment]
@hhkbp2 PTAL

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**